common/components/Header/components/Navigation.tsx

```diff
@@ -57,6 +58,7 @@ export default class Navigation extends PureComponent<Props, State> {
                 link={link}
                 isHomepage={link === navigationLinks[0]}
                 className="NavigationLink"
+                isNotEnabled={unsupportedTabs != null && unsupportedTabs.includes(link.name)}
```

Comment:
nitpick: `unsupportedTabs && unsupportedTabs.includes(link.name)` is more concise